functors_task_1/student/include/impl.h

```diff
@@ -8,6 +8,18 @@
  */
 struct BeerOrganizer
 {
+    size_t m_start = static_cast<size_t> (BeerBrand::HoeGaarden);
```

Comment:
1. Const
2. It is better to bind these values to a "special" entries like "Max" or "None"